app/views/play/SpectateView.coffee

```diff
@@ -204,7 +211,7 @@ module.exports = class SpectateLevelView extends RootView
     goldInDuelStatsView = @level.get('slug') in ['wakka-maul', 'cross-bones']
     @insertSubView new GoldView {} unless goldInDuelStatsView
     @insertSubView new HUDView {level: @level}
-    @insertSubView new DuelStatsView level: @level, session: @session, otherSession: @otherSession, supermodel: @supermodel, thangs: @world.thangs, showsGold: goldInDuelStatsView if @level.isLadder()
+    @insertSubView new DuelStatsView level: @level, session: @session, otherSession: @otherSession, supermodel: @supermodel, thangs: @world.thangs, anonymous: @anonymousPlayerName, me: me, showsGold: goldInDuelStatsView if @level.isLadder()
```

Comment:
any reason we need to pass me? Isn't it globally available?

Reply:
oh.. i can check. maybe it's global 😂  i dono